engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -644,6 +644,15 @@ protected void serviceStart() throws Exception
       if (SecurityUtils.isStramWebSecurityEnabled()) {
```

Comment:
Yes, I had the same block in mind. The initAuth is initializing a client utility. The class is meant for utility purposes and not a security manager. "Spreading that knowledge/functionality across multiple classes should be avoided." is inaccurate here.